extension/event-page.js

```diff
@@ -194,12 +222,33 @@ chrome.runtime.onMessage.addListener(
 chrome.runtime.onMessage.addListener(
   function (request, sender, sendResponse) {
 
-    if (request.method!='getAmKey') {
+    if (request.method!='getAmKeyAndManifests') {
       return;
     }
 
-    sendResponse(amKey);
-    return true;
+    sendResponse({amKey: amKey, manifestUrls: additionalManifestUrls});
   }
 
 );
+*/
+
+chrome.runtime.onMessage.addListener(
+  function(request, sender, sendResponse) {
+    if (request.method == 'storePageEntities') {
+      let key = request.url;
+      chrome.storage.local.set({[request.url]: request.results});
```

Comment:
How should we do error handling for chrome.storage.local calls here? How are we notified of errors from the set/get calls, and should we just console log for now?